mod.js

```diff
@@ -1,5 +1,7 @@
+/// <reference path="./types/index.d.ts" />
+
 export * from './lib/index.js'
 
 import postgres from './lib/index.js'
 
-export { postgres }
\ No newline at end of file
+export default postgres
```

Comment:
Removing `export { postgres }` is a breaking change. Both version can be kept though.
```suggestion
export default postgres
export { postgres }
```